openbas-front/src/admin/components/scenarios/scenario/ScenarioContext.ts

```diff
@@ -32,10 +35,14 @@ const injectContextForScenario = (scenario: ScenarioStore) => {
       return dispatch(deleteInjectScenario(scenario.scenario_id, injectId));
     },
     onImportInjectFromXls(importId: string, input: InjectsImportInput): Promise<void> {
-      return importXls(scenario.scenario_id, importId, input).then((_value) => new Promise((resolve, _reject) => {
+      return importXls(scenario.scenario_id, importId, input).then((response) => new Promise((resolve, _reject) => {
         dispatch(fetchScenarioInjects(scenario.scenario_id));
         dispatch(fetchScenario(scenario.scenario_id));
         dispatch(fetchScenarioTeams(scenario.scenario_id));
+        const criticalMessages = response.data.import_message.filter((value: ImportMessage) => value.message_level === 'CRITICAL');
```

Comment:
Should be put in scenario-action to show success or error depending on result

Reply:
Should be better now (although I'm unsure how clean the part in openbas-front/src/admin/components/common/injects/ImportUploaderInjectFromXls.tsx is)

Reply:
I don't understand what your mean